pkgs/tools/misc/autorandr/default.nix

```diff
@@ -1,22 +1,22 @@
 { fetchgit
 , stdenv
 , enableXRandr ? true, xrandr ? null
-, enableDisper ? false, disper ? null
+, enableDisper ? true, disper ? null
 , xdpyinfo }:
 
 assert enableXRandr -> xrandr != null;
 assert enableDisper -> disper != null;
 
 let
-  rev = "4f5e2401ef";
+  rev = "59f6aec0bb72e26751ce285d079e085b7178e45d";
 in
   stdenv.mkDerivation {
     name = "autorandr-${rev}";
```

Comment:
Git commit hashes should not be used as Nix versions, because they're not guaranteed to start with a digit, and are not monotonically increasing. This can confuse nix-env.